hw16/index.js

```diff
@@ -0,0 +1,151 @@
+'use strict'
+
+/*
+var admin;
+let userName;
+
+userName='Джон';
+admin=userName;
+
+console.log(admin);
+
+alert(admin)
+
+ourPlanet="Наша Планета"
+
+currentUserOfWebsite
+
+*/
+
+//TASK 1
+
+/*
+let x = 20;
+let y = 58;
+let z = 42;
+let plusResult = x + y + z
+
+console.log(plusResult)
+
+*/
+
+//TASK 2
+
+/*
+let a = 1;
+
+let minute = a * 60;
+let hours = minute * 60;
+let day = hours * 24;
+let year = day * 365;
+
+console.log(a);
+console.log(minute);
+console.log(hours);
+console.log(day);
+console.log(year);
+
+let myAgeInSecond = (27 * year);
+
+console.log(myAgeInSecond);
+
+*/
+
+//TASK 3
+
+/*
+let count = 42;
+let userName = '42';
+
+count = String(42);
+
+userName = Number(42);
+
+console.log(count);
+console.log(userName);
+
+*/
+
+//TASK 4
+
+/*
+let a = 1;
+let b = 2;
+let c = "белых медведей";
+
+a = String(1);
+b = String(2);
+
+console.log(a + b + c);
+
+*/
+
+//TASK 5
+
+/*
+let a = 'доступ';
+let b = 'морпех';
+let c = 'наледь';
+let d = 'попрек';
+let e = 'рубило';
+
+console.log(a.length + b.length + c.length + d.length + e.length);
+
+*/
+
+//TASK 6 
+
+/*
+let a = String('`Variable: %variable name% have type: %type variable%`');
+let b = Number('`Variable: %variable name% have type: %type variable%`');
+let c = Symbol('`Variable: %variable name% have type: %type variable%`');
+
+const d = '`Variable: %variable name% have type: %type variable%`';
+
+a = d;
+b = d;
+c = d;
```

Comment:
В конце присвоил d, так как выскакивала ошибка? :)

Решение проще чем ты думаешь
```suggestion
let a = String('`Variable: %variable name% have type: %type variable%`');
let b = Number(10);
let c = Symbol('`Variable: %variable name% have type: %type variable%`');

console.log(`Variable: a have type: ${typeof a}`)
console.log(`Variable: b have type: ${typeof b}`)
console.log(`Variable: c have type: ${typeof c}`)
```